README.md

```diff
@@ -145,7 +147,13 @@ the calls to each spied function and the arguments passed to them in each call.
     (is (= ["koko" "triki"] (-> some-function td/calls-to first)))
 
     (is (= 3 (-> println td/calls-to count)))
-    (is (->> greetings-function td/calls-to (every? #(= % "Hola!"))))))
+    (is (->> println td/calls-to (every? #(= % ["Hola!"]))))
+
+    (try
+      (td/calls-to greetings-function)
+      (catch :default e
+        (is (= "Checking calls to not spied function"
```

Comment:
I think the message might read better as "Attempting to check calls for a function that is not being spied on"